### DISCUSSION TITLE
Can I enable diferent models per user?

### DISCUSSION THREAD

**Discussion by miesgre** [open-webui/open-webui#2733] [Q&A]
I am using openwebui with litellm and I only have some of the Ollama models enabled in openwebui, but I want to give some of my users access to other models (Azure/OpenAI that are pay-per-use). Can I enable these models only for a group of users?

Thanks

**Comment by JuanjoJG-WATA** [comment]
You can whitelist different models from **Admin Panel > Admin Settings > Users**

**Reply by miesgre** [reply]
Yes, I know and I am using it. But what I want is for some users (ex. user-123, and user-345) to also have access to other models (normally paid models).

**Reply by JuanjoJG-WATA** [reply]
I think they are working in groups at the moment, maybe for v0.5, but I'm not completely sure.

**Reply by miesgre** [reply]
great! thank you

**Reply by SUNKENDREAMS** [reply]
Bump: This would still be a great capability to have. Creating 'new models' and assigning them to specific users per their use-case or professional needs. Helps to improve providing LLMs as a service.

**Reply by lynxul25** [reply]
Vote from my side - it would allow for different type of users to be able to use the platform in a controlled way (kids, adults, developers, etc.)

**Reply by antonio-lamberti** [reply]
+1 for feature request!

**Reply by herrids** [reply]
I would need the feature to decide which users use which model so bad. Working with groups would be amazing

**Reply by MrRedtastic** [reply]
+1 on User Groups and Whitelisting that way

**Reply by olipericles** [reply]
+1 for feature request!

**Reply by Richie3479** [reply]
+1 for feature request!

**Comment by Chinaduanyun** [comment]
yesssss!I need this

**Comment by ngtrphuong** [comment]
+1 for this feature request in Apr 2025,

**Reply by tjbck** [reply]
We've had User Groups feature for a long time.

**Reply by adhusch** [reply]
But how can one assign different models to different groups? At the moment it seems all groups always see **all** models or **no** models but no models-per-group setting is available?

**Reply by Classic298** [reply]
admin settings > models > edit model > private but allow for specific group

**Reply by adhusch** [reply]
Thank you 👍

**Comment by Classic298** [comment]
@miesgre please mark the discussion as answered so others can find it more easily when looking for a solution 👍
